packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -1056,12 +1070,14 @@ export function TransferPanel() {
                 }
               }}
               style={{
+                borderColor: transferReady.deposit
```

Comment:
Applies to deposits and withdrawals.

Shouldn't this always be `sourceChainUIcolor.primary`? Otherwise e.g. the color changes between source/destination based on whether I insert amount in the input field or not.

E.g. I have Xai <> Arb One selected. Then:
- if amount input empty, the border is red
- if amount input populated, the border is blue

May be easier to not have border color if the button is disabled? May be simpler and more consistent.

Reply:
i fixed it to grey, yea it wasn't meant to be the source chain colour ever oops

but it needed the border colour to maintain a consistent height for the button